code/PostProcessing/ValidateDataStructure.cpp

```diff
@@ -371,6 +371,7 @@ void ValidateDSProcess::Validate(const aiMesh *pMesh) {
         ReportWarning("There are unreferenced vertices");
     }
 
+#if 0 // no documentation found on why is it necessary
```

Comment:
Could you remove the code, please?

Reply:
I removed 3 unwanted commits and that #if 0 block.
It's first time I'm using hard reset on public branch, is this how things should be done?

Reply:
You can just change it on your branch. We will merge this as one commit. So no hard reset will be necessary.